10-angular/custom_directives/readme.md

```diff
@@ -679,14 +682,6 @@ directives.directive('grumbleSave',[
 );
 ```
 
-This means I can delete the newGrumbleController altogether. So I'll delete that from `js/controllers/grumbles.js`, and I'll remove it from `routes.js`:
-
-```js
-when("/grumbles/new", {
-  templateUrl: 'views/grumbles/new.html'
-}).
-```
-
 ...and when I try to save a Grumble, it works!
```

Comment:
Add a `You Do` section for implementing update functionality in the form directive.